tools/eval/README.md

```diff
@@ -211,4 +212,18 @@ Run evaluation on quantized model with the following command :
 
 ```shell
 python eval.py -m pphumanseg_q -d mini_supervisely -dr /path/to/pphumanseg
-```
\ No newline at end of file
+```
+
+## OTB
+
+### Prepare data
+
+Please visit [here](https://drive.google.com/drive/folders/1DZvtlnG9U94cgLD6Yi3eU7r6QZJkjdl-?usp=sharing) to download the OTB dataset and the json file. Please both files together in a directory.
```

Comment:
Do you have the official link to download the dataset from the official OTB website?

Reply:
Also https://paperswithcode.com/dataset/otb mentions that OTB has `-2015` and `-2013` and `-50` variances. Which variance is used here?

Reply:
@fengyuentau From https://paperswithcode.com/dataset/otb:
[OTB-2013](https://paperswithcode.com/dataset/otb-2013) dataset contains 51 sequences and the [OTB-2015](https://paperswithcode.com/dataset/otb-2015) dataset contains all 100 sequences of the OTB dataset.
The official linked website http://cvlab.hanyang.ac.kr/ is down there therefore I couldn't use a direct link from the official OTB website.

I am currently using the 2015 version because it contains all 100 sequences.

I also found a different way to download each individual sequence through this archived version of the official website:
https://web.archive.org/web/20220210102816/http://cvlab.hanyang.ac.kr/tracker_benchmark/datasets.html
If this is preferred, I can update the PR to download from here instead.

Reply:
So I propose to change dataset name to OTB-2015 or OTB-100 for presentation and `otb2015` or `otb100` in the code.

---

> The official linked website http://cvlab.hanyang.ac.kr/ is down, therefore I couldn't use a direct link from the official OTB website.

In this case, you should change the description saying that

1. The official site is http://cvlab.hanyang.ac.kr/.
2. In case it is down, users can download from the alternative link we provide.

I am now downloading the dataset and uploading to our google drive so that it won't be missed in the future. The link should be replaced with the one that I will provide later.

Reply:
The link is https://drive.google.com/drive/folders/1iTwCQAMgzdWWrlwncOjpshuHvipIWPMN?usp=sharing

Reply:
I've updated it with the link. I like how @lpylpy0514 pulls the relevant information within the original dataset instead of relying on a specially made JSON.
I plan to update it with that functionality in place of modifying the directory structure after figuring out the AUC.